### DISCUSSION TITLE
Force flux to redo failed helm upgrade?

### DISCUSSION THREAD

**Discussion by mwineland587** [fluxcd/flux2#1764] [Q&A] [answered]
Using the bitnami wordpress helm packages and an upgrade failed because the PVC already existed. I updated the flux manifest to include the name so that it would work, but helm refuses to continue now:

 helmVersion=v3 error="failed to determine sync action for release: status 'failed' of release does not allow a safe upgrade"

We have a lot of customization in our flux manifest, so adding flags for every env with a helm command seems daunting, is there a recommended way to fix this?

**Comment by hiddeco** [comment, marked as answer]
See the Helm Operator FAQ: https://fluxcd.io/legacy/helm-operator/faq/#a-helmrelease-is-stuck-in-a-failed-release-state-how-do-i-force-it-through

Please note however that this discussion board is about Flux version 2, and we are encouraging users to move away from "legacy Flux" and the Helm Operator.
